Sources/Extensions/UIView+Constraints.swift

```diff
@@ -8,14 +8,50 @@
 import UIKit
 
 extension UIView {
+    /// Constrain this view to the it's parent view.
+    ///
+    /// - parameter insets: The insets that give a some padding if needed.
+    public func constraintToSuperview(insets: UIEdgeInsets = .zero) {
+        // A superview should be set before setting constraints.
+        guard let superview = superview else { return }
+        constraintEdges(to: superview, insets: insets)
+    }
     
-    public func constraintEdges(to view: UIView) {
+    /// Constrain this view to the given view.
+    ///
+    /// - parameter view: The view to constraint it's edges to.
+    /// - parameter insets: The insets that give a some padding if needed.
+    public func constraint(to view: UIView, insets: UIEdgeInsets = .zero) {
         translatesAutoresizingMaskIntoConstraints = false
         
-        leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
-        rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
-        topAnchor.constraint(equalTo: view.topAnchor).isActive = true
-        bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        topAnchor.constraint(equalTo: view.topAnchor, constant: insets.top).isActive = true
```

Comment:
Kwestie van smaak: als ik alles op `active` moet zetten, gebruik ik vaak

```
NSLayoutConstraint.activate([
   topAnchor.constraint(equalTo: view.topAnchor, constant: insets.top),
   rightAnchor.constraint(equalTo: view.rightAnchor, constant: -insets.right),
   bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -insets.bottom),
   leftAnchor.constraint(equalTo: view.leftAnchor, constant: insets.left)
])
 ```

Maar never mind 😉

Reply:
Zouden we ook geen `leading` en `trailing` gebruiken hiervoor?

Reply:
Ik fix beide comments!